diff --git a/node_modules/@capacitor/local-notifications/android/src/main/kotlin/com/capacitorjs/plugins/localnotifications/LocalNotificationManager.kt b/node_modules/@capacitor/local-notifications/android/src/main/kotlin/com/capacitorjs/plugins/localnotifications/LocalNotificationManager.kt
--- a/node_modules/@capacitor/local-notifications/android/src/main/kotlin/com/capacitorjs/plugins/localnotifications/LocalNotificationManager.kt
+++ b/node_modules/@capacitor/local-notifications/android/src/main/kotlin/com/capacitorjs/plugins/localnotifications/LocalNotificationManager.kt
@@ -192,6 +192,7 @@
             mBuilder.setGroup(group)
             if (localNotification.groupSummary) {
                 mBuilder.setSubText(localNotification.summaryText)
+                mBuilder.setGroupAlertBehavior(NotificationCompat.GROUP_ALERT_CHILDREN)
             }
         }
 
